runtimes/alibi-explain/mlserver_alibi_explain/common.py

```
@@ -37,10 +37,15 @@ def convert_from_bytes(output: ResponseOutput, ty: Optional[Type]) -> Any:
     This utility function decodes the response from bytes string to python object dict.
     It is related to decoding StringCodec
     """
+    if output.shape != [1]:
+        # TODO: Does this default value make sense here?
+        return ""
```

should we raise an exception?